app/controllers/home_controller.rb

```diff
@@ -95,10 +95,10 @@ def dashboard
     @comments = @comments.group('DATE(FROM_UNIXTIME(timestamp))') if Rails.env == "production"
     @answer_comments = DrupalComment.joins(:answer, :drupal_users)
                              .order('timestamp DESC')
-                             .where('timestamp - answers.created_at > ?', 0)
+                             .where('timestamp - answers.created_at > ?', 86400)
```

Comment:
It's the same value used in comments. It is the time difference between the answer creation and comment creation before which it doesn't gets published. Similar to this: https://github.com/publiclab/plots2/blob/master/app/controllers/home_controller.rb#L90. Actually I thought of skipping this off as it is mentioned just before this.